config/i18n.json

```diff
@@ -1335,6 +1336,7 @@
     "Untitled": "Untitled Wish List",
     "Voltron": "voltron (default)",
     "VoltronDescription": "voltron is an auto-updated collection of PvE and PvP wish list rolls from across the Destiny community. We ship with this by default.",
+    "VoltronSelected": "voltron (default) selected.",
```

Comment:
Localization strings support interpolation -- I'd prefer to have a string like `{{wishlistName}} selected.` and then use it like `t('WishListRoll.SelectedList', { wishlistName: t('WishListRoll.Voltron')})`. Otherwise LGTM, I agree with the reasoning in the linked issue.

<sup>use `Fixes #9421` in the PR description to automatically close the issue when this is merged</sup>